tests/integration/components/frost-select-test.js

```diff
@@ -182,173 +172,147 @@ describeComponent(
-      let input = this.$('.frost-select input')
-      keyUp(dropDown, 'down')
-      keyUp(dropDown, 13) // Enter key, select the item
-
-      run.later(() => {
+    it('sets the prompt to the selected value when the drop down list is closed', function () {
+      run(() => {
+        let input = this.$('.frost-select input')
+        keyUp(dropDown, 'down')
+        keyUp(dropDown, 13) // Enter key, select the item
         input.val('')
         keyUp(dropDown, 'down')
         keyUp(dropDown, 'esc')
-
-        run.later(() => {
-          let select = this.$('.frost-select')
-          let input = this.$('.frost-select input')
-          expect(select.hasClass('open')).to.be.false
-          expect(input.val()).to.eql('Raekwon')
-          done()
-        })
       })
+
+      let select = this.$('.frost-select')
+      let input = this.$('.frost-select input')
+      expect(select.hasClass('open')).to.be.false
+      expect(input.val()).to.eql('Raekwon')
     })
 
-    it('handles click outside of select', function (done) {
-      this.$('.frost-select .down-arrow').click()
-      run.later(() => {
+    it('handles click outside of select', function () {
+      run(() => {
+        this.$('.frost-select .down-arrow').click()
         this.$().click()
-        run.later(() => {
-          expect(dropDown.hasClass('open')).to.be.false
-          done()
-        })
       })
+      expect(dropDown.hasClass('open')).to.be.false
     })
 
-    it('handles loosing focus by pressing tab', function (done) {
-      this.$('.frost-select .down-arrow').click()
-      run.later(() => {
+    it('handles loosing focus by pressing tab', function () {
+      run(() => {
+        this.$('.frost-select .down-arrow').click()
         keyDown(dropDown, 'tab')
-        run.later(() => {
-          expect(dropDown.hasClass('open')).to.be.false
-          done()
-        })
       })
+      expect(dropDown.hasClass('open')).to.be.false
     })
 
     it('supports placeholder', function () {
       const $input = this.$('.frost-select input')
       expect($input.attr('placeholder')).to.eql('Select something already')
     })
-  }
-)
-
-describeComponent(
-  'frost-select',
-  'Integration: FrostSelectComponent',
-  {
-    integration: true
-  },
-  function () {
-    let props
 
-    beforeEach(function () {
-      props = {
-        onChange: sinon.spy(),
-        placeholder: 'Select something already',
-        data: [
-          {
-            value: 'Lex Diamond',
-            label: 'Raekwon'
-          },
-          {
-            value: 'Johnny Blaze',
-            label: 'Method Man'
-          },
-          {
-            value: 'Tony Starks',
-            label: 'Ghostface'
-          }
-        ],
-        greeting: 'Hola'
-      }
-      this.setProperties(props)
-      this.render(selectedValueTestTemplate)
-    })
+    describe('when passing in selected value', function () {
```

Comment:
Oh yeah, there were two `describeComponent` calls in the one module, which seemed off to me.  Since they were for the same component anyway, I just made the latter into a `describe()` within the first `describeComponent`